internal/command/console_interactive.go

```diff
@@ -68,7 +68,7 @@ func (c *ConsoleCommand) modeInteractive(session *repl.Session, ui cli.Ui) int {
 				c.showDiagnostics(diags)
 			}
 			if exit {
-				break
+				return 0
```

Comment:
nit: I would like to change other break statements to returns just in case someone puts something after the loop in the future and doesn't account for this exact return.